fetch_system_config/debian/fetch-noetic-config.postinst

```diff
@@ -60,7 +60,7 @@ case "$1" in
         fi
 
         # For PS4 controller support, install ds4drv with pip, presuming that pip is installed already
-        pip install ds4drv
+        /usr/local/bin/pip install ds4drv
```

Comment:
How confident are we this is the path?

Reply:
I had to do some reading.  /usr/local/bin is where our pip ends up since we use get-pip.py.  /usr/local/bin is used instead of /usr/bin when installing without using the system's package manager (i.e. `apt`).

I verified this on multiple 20.04 systems.

But yeah... why?

Hmmm. I should change the top level to /usr/bin/bash from /bin/bash, too... https://askubuntu.com/questions/1242157/why-are-bin-and-sbin-now-symlinks-in-ubuntu-20-04

Reply:
Changing the shebang didn't remove the need for this line's change. Oh well. All the invocations of stuff at /bin now properly reference /usr/bin.